public/js/main.js

```diff
@@ -0,0 +1,66 @@
+var storageKeyAll = 'allList';
+var storageKeyTodo = 'todoList';
+var storageKeyDone = 'doneList';
+
+var todoList = [];
+var allList = [];
+var doneList = [];
```

Comment:
Can we use only todoList?

Reply:
I think that maybe I could use them when creating functions for showing active tasks and complete tasks

Reply:
correct, you should use one list for all, filter by `done` flag

Reply:
I have changed to use todoList and doneList for storing filters of allList